tests/Aggregation/RangeTest.php

```diff
@@ -88,11 +88,13 @@ protected function _getIndexForTest(): Index
         $index = $this->_createIndex();
 
         $index->addDocuments([
+            // this is not a mistake, dynamic mapping will take first type
```

Comment:
Make sense, that would also explain why on line 28 it was 2 before, so 1 doc got dropped I guess. Ideally we would load the mapping ourselfs instead of relying on dynamic mappings. Glad you found this.